demo.py

```diff
@@ -0,0 +1,29 @@
+import minedojo
+from wrappers import MineClipWrapper
+
+
+def run():
+    env = minedojo.make(
```

Comment:
What if we use this task as our go to: https://github.com/MineDojo/MineDojo/blob/d0691f2bdbb18dd4885d1ef251ac29c25786127b/minedojo/tasks/description_files/programmatic_tasks.yaml#L6193

Reply:
Also, can we set a horizon in the make function so that you don't run into your it's been running forever problem? Or we can make another wrapper for that.

Reply:
I will merge for now and try out with the suggested tasks this week!